riscv/encoding.h

```diff
@@ -421,6 +421,12 @@
 #define MASK_AMOAND_D 0xf800707f
 #define MATCH_AMOAND_W 0x6000202f
 #define MASK_AMOAND_W 0xf800707f
+#define MATCH_AMOCAS_D 0x2800302f
+#define MASK_AMOCAS_D 0xf800707f
+#define MATCH_AMOCAS_Q 0x2800402f
+#define MASK_AMOCAS_Q 0xf800707f
+#define MATCH_AMOCAS_W 0x2800202f
+#define MASK_AMOCAS_W 0xf800707f
```

Comment:
I assume those are side effects of the `riscv-opcodes` update (and not directly related to vector crypto) ?

Reply:
Indeed. The encoding update is the one from https://github.com/riscv-software-src/riscv-isa-sim/pull/1303. It is made from riscv-opcode's master branch at HEAD because the previous encoding update contained commits very close to (at?) HEAD but seemingly not from the master branch or any commit that appears in riscv-opcodes's repo.